tests/apps/borg/test_queen.py

```diff
@@ -12,16 +10,20 @@
 __copyright__ = "Copyright 2012, The Materials Project"
 __date__ = "Mar 18, 2012"
 
+TEST_DIR = f"{TEST_FILES_DIR}/app_borg"
+
 
 class TestBorgQueen(unittest.TestCase):
     def test_get_data(self):
+        """Test get data from vasprun.xml.xe.gz file."""
         drone = VaspToComputedEntryDrone()
-        self.queen = BorgQueen(drone, TEST_FILES_DIR, 1)
-        data = self.queen.get_data()
-        assert len(data) == 16
+        queen = BorgQueen(drone, TEST_DIR, 1)
+        data = queen.get_data()
+        assert len(data) == 1
+        assert data[0].energy == 0.5559329
```

Comment:
I should have used `approx` here and I would make this change in a future PR somewhere.